app/renderer/components/settings.js

```diff
@@ -58,10 +59,20 @@ class SettingCheckbox extends ImmutableComponent {
   }
 
   onClick (e) {
+    if (this.props.forPassword) {
+      // You can only have one active password manager
+      // if user decide to disable all,
+      // switch back password to unmanaged (null)
+      e.target.value
+        ? aboutActions.changeSetting(settings.ACTIVE_PASSWORD_MANAGER, this.props.prefKey)
+        : aboutActions.changeSetting(settings.ACTIVE_PASSWORD_MANAGER, null)
```

Comment:
Similar pattern in _extensionsTab.js_. There `false` was used; here `null` is being used. Can we go with one or the other, so there isn't any confusion about whether or not the two behave differently?

```
+    isPasswordManager(extensionId)
+      ? aboutActions.changeSetting(settings.ACTIVE_PASSWORD_MANAGER, extensionId)
+      : aboutActions.changeSetting(getExtensionSetting(extensionId), false)
```